fixtures/schema.js

```diff
@@ -147,6 +148,11 @@ const Fixture = schema({
   }),
   '?comment': NonEmptyMultiLineString,
   '?manualURL': URL,
+  '?rdm': schema({
+    'modelId': Number.min(0).step(1),
```

Comment:
Max is defined for this 0xffff it should be (I'll double check the standard later).